salesforce-connector-demo/processes/ChartDemo.p.json

```diff
@@ -0,0 +1,47 @@
+{
+  "format" : "10.0.0",
+  "id" : "18C19135AC79638F",
+  "config" : {
+    "data" : "salesforce.connector.demo.Data"
+  },
+  "elements" : [ {
+      "id" : "f0",
+      "type" : "RequestStart",
+      "name" : "start.ivp",
+      "config" : {
+        "callSignature" : "start",
+        "outLink" : "start.ivp"
+      },
+      "visual" : {
+        "at" : { "x" : 96, "y" : 64 }
+      },
+      "connect" : { "id" : "f3", "to" : "f2" }
+    }, {
+      "id" : "f1",
+      "type" : "TaskEnd",
+      "visual" : {
+        "at" : { "x" : 472, "y" : 64 }
+      }
+    }, {
+      "id" : "f2",
+      "type" : "UserTask",
+      "name" : "Chart Demo",
+      "config" : {
+        "dialogId" : "salesforce.connector.demo.Chart",
+        "startMethod" : "start()",
+        "case" : {
+          "name" : "Chart Demo",
+          "description" : "Chart Demo"
+        },
+        "task" : {
+          "name" : "Chart Demo",
```

Comment:
Could we define the task/case/process name as the guideline here: https://1ivy.atlassian.net/wiki/spaces/AIGUIDE01/pages/2918379/Content+Management+System+Organization
@HoDacVu90